lib/cms/hippo_importer/base.rb

```diff
@@ -0,0 +1,86 @@
+module Cms
+  module HippoImporter
+    class Base
+      attr_reader :records
+
+      def initialize(data:, docs: [], to: 'en', parser: HippoXmlParser)
+        @records = parser.parse(data, [hippo_type])
+        @_site = to
+        @docs = docs
+      end
+
+      def hippo_type
+        'contentauthoringwebsite:Guide'
+      end
+
+      def site
+        @site ||= Comfy::Cms::Site.find_by(label: @_site)
```

Comment:
I'd move this to `initialize` and then you don't need to mess around with the whole `@_site` and `@site` stuff.